### PR TITLE
[PWA-616] [Mini-Cart] Swatch images are stretched in mini-cart and has…

### PR DESCRIPTION
**Pull request by anthoula**
## Description

Ensure mini cart swatch images are not stretched and aligned to top of product info text

## Related Issue

https://jira.corp.magento.com/browse/PWA-616

### Verification Stakeholders

@jimbo 
@dhaecker 

### Verification Steps

1. Add a product to cart with a wide image
2. Go to mini cart page and view thumbnail image is not stretched

## Checklist

- [x] I have added tests to cover my changes, if necessary.
- [x] I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Comment by devops-pwa-codebuild** [comment]
**Performance Test Results**

The following fails have been reported by WebpageTest. These numbers indicates a possible performance issue with the PR which requires further manual testing to validate.

https://pr-2457.pwa-venia.com : LH Performance Expected 0.85 Actual 0.57, LH Best Practices Expected 1 Actual 0.92
https://pr-2457.pwa-venia.com/venia-tops.html : LH Performance Expected 0.75 Actual 0.33, LH Best Practices Expected 1 Actual 0.92
https://pr-2457.pwa-venia.com/valeria-two-layer-tank.html : LH Performance Expected 0.8 Actual 0.49, LH Accessibility Expected 0.9 Actual 0.89, LH Best Practices Expected 1 Actual 0.92

File: packages/venia-ui/lib/components/SearchBar/suggestedProduct.css [hunk 1]
```diff
@@ -15,6 +15,8 @@
 .thumbnail {
     height: 75px;
     max-height: 75px;
+    object-fit: contain;
+    object-position: center;
```
Comment (anthoula) [inline packages/venia-ui/lib/components/SearchBar/suggestedProduct.css:19]: This is centered to align with vertically centered product test.

File: packages/venia-ui/lib/components/CartPage/ProductListing/product.css [hunk 2]
```diff
@@ -13,6 +13,9 @@
     background-color: rgb(var(--venia-grey));
     border: solid 1px rgb(var(--venia-border));
     border-radius: 2px;
+    height: 100%;
+    object-fit: contain;
+    object-position: center;
```
Comment (anthoula) [inline packages/venia-ui/lib/components/CartPage/ProductListing/product.css:18]: Small change from UX, vertically center all images

Comment (jimbo) [reply packages/venia-ui/lib/components/CartPage/ProductListing/product.css:18]: That should work better. 👍

**Comment by dhaecker** [comment]
QA approved

**Pull request merged**